src/java/com/wolfssl/provider/jsse/WolfSSLContext.java

```diff
@@ -458,6 +458,11 @@ protected void finalize() throws Throwable {
         }
         super.finalize();
     }
+    
+    public String[] getProtocolsMask(long noOpt) {
+            ctx.setOptions(noOpt);
```

Comment:
We should handle the case where "ctx" is null here.  Consider either returning null or throwing a IllegalStateException similar to how other methods in WolfSSLContext.java do.

Reply:
fixed